emscripten.py

```diff
@@ -2360,6 +2360,28 @@ def create_asm_consts_wasm(forwarded_json, metadata):
       all_sigs.append((sig, call_type))
 
   asm_const_funcs = []
+  asm_const_funcs.append(r'''
+function _read_asm_const_args(sig_ptr, buf) {
```

Comment:
as a pure JS helper, how about `readAsmConstArgs`?

Reply:
Sure. Underscore prefix to avoid userspace conflicts? I don't think these are in Module, are they?

Also this is a pretty dumb place to put this helper, it doesn't depend on being generated dynamically, should it go in preamble.js?

Reply:
In general we haven't underscored our JS API stuff. I think it's fine without.

I think it's ok to emit this here, but actually I had assumed it was only emitted if there was at least one EM_ASM, which re-reading now I see is not the case. I think adding that check, and keeping it here (where we know if there are EM_ASMs) would be best.